packages/styles/scss/themes/expressive/components/_modal.scss

```diff
@@ -0,0 +1,25 @@
+//
+// Copyright IBM Corp. 2016, 2018
+//
+// This source code is licensed under the Apache-2.0 license found in the
+// LICENSE file in the root directory of this source tree.
+//
+
+@import '@carbon/layout/scss/convert';
+
+//-----------------------------
+// Modal (expressive)
+//-----------------------------
+
+/// Modal styles (expressive)
+/// @access private
+/// @group modal-expressive
+@mixin modal-expressive {
+  .#{$prefix}--modal-content--overflow-indicator {
+    background-image: linear-gradient(
+      to bottom,
+      rgba(244, 244, 244, 0),
+      #f4f4f4
```

Comment:
can we use a color-token here instead?

Reply:
looks like in carbon, they have `background-image: linear-gradient(to bottom, rgba($ui-01, 0), $ui-01);` in `_modal.scss` 

have you tried updating to the latest carbon dependencies to see if this issue would be resolved?

Reply:
Yeah updating carbon dependencies doesn't fix the issue